utilities/EluvioLiveCli.js

```diff
@@ -44,6 +44,24 @@ const Init = async ({debugLogging = false, asUrl}={}) => {
   await marketplace.Init({ debugLogging });
 };
 
+const CmdTenantAuthToken = async ({ argv }) => {
+  await Init({debugLogging: argv.verbose});
+
+  let ts = Date.now();
+  let message = argv.path_or_body + "?ts=" + ts
+  try {
+    let j = JSON.parse(argv.path_or_body)
+    j.ts = ts;
+    message = JSON.stringify(j)
+  } catch (e) {}
+
+  const { multiSig } = await elvlv.TenantSign({
+    message: message
+  });
+  console.log(message)
+  console.log(`Authorization: Bearer ${multiSig}`);
```

Comment:
Can we return a YAML that can have message and token?  The other commands return YAML (which is both human readable-ish and command line parseable).

Reply:
```
~/e/git/elv-live-js $ ./elv-live tenant_auth_token '{"hello":123}'
Network: demov3
Timestamped path or body: {"hello":123,"ts":1689024549073}
Token: Authorization: Bearer ES256K_LtRWGcHG3y9EiRpMu7YKU8doYXmSSXPGgGfpbUjq8PKyMdXrD9Ngn9HhoxARMhovogzG4FnUvwapRyQSZMmzDrM2B
~/e/git/elv-live-js $ ./elv-live tenant_auth_token "/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a"
Network: demov3
Timestamped path or body: /nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a?ts=1689025512273
Token: Authorization: Bearer ES256K_En3WkxJQBgHuLazoZ9NYMkEWiLg9M3vfesccRtd3sdACMzfG3tvEhiA2Xtrqwr74WHqKtHRXJngvdBSEevRvhTxAf
```

Reply:
not sure if this is valid YAML with the extra colons